Emel/index.html

```diff
@@ -0,0 +1,259 @@
+
+    <!-- <style>
+      .bd-placeholder-img {
+        font-size: 1.125rem;
+        text-anchor: middle;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        user-select: none;
+      }
+
+      @media (min-width: 768px) {
+        .bd-placeholder-img-lg {
+          font-size: 3.5rem;
+        }
+      }
+    </style> -->
+
+    
+    <!-- Custom styles for this template -->
+    <link href="form-validation.css" rel="stylesheet">
+  </head>
+  <body class="bg-light">
+ 
+    <main>  
+    <div class="container m-5 p-5" style="position: absolute; left: 10%;">
+      <form action="POST">
+
+      </form>
+      <h4 class="mb-3">Registration Details</h4>
+          <br>
+      <div class="row">
+        <div class="col-sm-1" style="align-content: center;">
+          
+          <!--Salutation & Name-->
+          <strong>
+            Name:
+          </strong>
+        </div>
+        <div class="col-sm-2">
+          <select class="form-select" id="salutation" required>
+            <option value="Mr.">Mr.</option>
+            <option value="Ms.">Ms.</option>
+            <option value="Dr.">Dr.</option>
+            <option value="Prof.">Prof.</option>
+          </select>
+        </div>
+        <div class="col-sm-3">
+          <input type="text" class="form-control" name="firstName" id="firstName" placeholder="First Name*" required>
+        </div>
+        <div class="col-sm-3">
+          <input type="text" class="form-control" name="middleName" id="middleName" placeholder="Middle Name" required>
+        </div>
+        <div class="col-sm-3">
+          <input type="text" class="form-control" name="lastName" id="lastName" placeholder="Last Name*" required>
+        </div>
+      </div>
+      <br>
+      <!--Email& Phone-->
+      <div class="row">
+        <div class="col-sm-3" style="align-items: flex-start;">
+          Email*:
+        </div>
+        <div class="col-sm-3" style="align-items: flex-start;">
+          <input type="email" class="form-control" id="email" placeholder="you@example.com">
+        </div>
+        <div class="col-sm-3">
+          Phone*:
+        </div>
+        <div class="col-sm-3">
+          <input type="tel" class="form-control" id="phone" placeholder="">
+        </div>
+      </div>  
+
+      <br>
+
+      <!--Affiliation-->
+      <div class="col-sm">
+        Affiliation*:
+      </div>
+      <div class="col-sm">
+        <input type="text" class="form-control" id="affiliation" placeholder="" value="" required>
+      </div>
+
+      <br>
+      
+      <!--Correspondence Address-->
+      <div class="col-sm">
+        Correspondence Address*:
+      </div>
+      <div class="col-sm">
+        <input type="address" class="form-control" id="correspondence_address" placeholder="" value="" required>
```

Comment:
remove empty placeholder and value tags.
there is no input type address. Refer to https://www.w3schools.com/html/html_form_input_types.asp for the different input types available